Workbooks/Availability/Downtime Outages/Downtime Outages.workbook

```diff
@@ -1450,14 +1441,20 @@
               "showAnalytics": true,
               "title": "Failures by Testing Location",
               "timeContext": {
-                "durationMs": 259200000
+                "durationMs": 0
               },
               "timeContextFromParameter": "TimeRange",
+              "exportMultipleValues": true,
+              "exportAggregateParts": true,
+              "exportedParameters": [
+                {
+                  "fieldName": "location",
+                  "parameterName": "mapLocations",
+                  "parameterType": 8
+                }
+              ],
               "queryType": 0,
               "resourceType": "microsoft.insights/components",
-              "crossComponentResources": [
-                "{Resources}"
-              ],
```

Comment:
abunch of places here have removed the selected parameter which is going to break them. not sure what/how this came in but isn't good

Reply:
Added back to match Kais original version